caffe2/utils/conversions.h

```diff
@@ -21,218 +21,16 @@ namespace caffe2 {
 
 namespace convert {
 
-namespace {
-inline float16 cpu_float2half_rn(float f) {
```

Comment:
If you want to be really careful about this, cross-test this with the ATen implementation to check if either have bugs :>